packages/utilities/psammead-test-helpers/README.md

```diff
@@ -0,0 +1,36 @@
+# psammead-test-helpers &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-test-helpers.svg)](https://www.npmjs.com/package/@bbc/psammead-test-helpers) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://reactjs.org/docs/how-to-contribute.html#your-first-pull-request)
+
+This package is a collection helper methods used for snapshot testing, with support for styled components.
+
+## Installation
+
+```jsx
+npm install @bbc/psammead-test-helpers --save
```

Comment:
To be able to use `psammead-test-helpers`, a consumer would need to also install `react react-dom react-test-renderer`.